File: Javascript/7-Kyu/All-star-code-challenge-#1.js
// All star code challenge #1
function sumPPG(playerOne, playerTwo){

    return playerOne.ppg + playerTwo.ppg;

};

// Test
function NBAplayer(name, team, ppg){
    this.name=name;
    this.team=team;
    this.ppg=ppg;
};

var iverson = new NBAplayer("Iverson", "76ers", 11.2);
var jordan = new NBAplayer("Jordan", "bulls", 20.2);

console.log(sumPPG(iverson, jordan));  // --> 31.4

File: Python/7-Kyu/All-star-code-challenge-#1.py
# All star code challenge #1
def sum_ppg(player_one, player_two):

    return player_one['ppg'] + player_two['ppg']


# Test

iverson = { 'team': '76ers', 'ppg': 11.2 }
jordan =  { 'team': 'bulls', 'ppg': 20.2 }

print(sum_ppg(iverson, jordan))  # --> 31.4
